_posts/2021-7-8-How-to-create-a-blazor-webassembly-grpc-web-app-using-app-service-on-an-azure-arc-enabled-kubernetes-cluster.md

```diff
@@ -0,0 +1,61 @@
+    - kubernetes
+---
+
+
+With the release of the [App Service on Kubernetes preview](https://azure.microsoft.com/updates/public-preview-run-app-service-on-kubernetes-or-anywhere-with-azure-arc/), you can now run App Service on Kubernetes deploying your web apps to Azure Kubernetes Service, or the cluster of your choosing. This enables you to utilize App Service features including continuous deployment with GitHub Actions and deployment slots. 
+
+In this tutorial, we'll be deploying a .NET 5 gRPC-Web app to a Custom Location (App Service deployed on an Arc-enabled AKS cluster) with App Service on Kubernetes using the same gRPC-Web app we created in [this blog](https://azure.github.io/AppService/2021/03/15/How-to-use-gRPC-Web-with-Blazor-WebAssembly-on-App-Service.html). This example assumes you already have your resource group, connected Arc cluster, custom location, and App Service Kubernetes environment setup. Follow the steps in the [documentation](https://docs.microsoft.com/azure/app-service/manage-create-arc-environment) if you do not have these resources created.
+
+## Creating your Web App
+Creating your web app resource in the Azure portal should look very familiar to what you've seen before.  When selecting your Resource Group, make sure it is the resource group you've created with all of your Azure Arc resources in them. Once the correct resource group is selected, as well as the operating system (Linux) you will find your Arc custom location in the Region drop-down menu.
+
+If you would like to use the CLI to create your App Service resource, please see the [documentation](https://docs.microsoft.com/azure/app-service/quickstart-arc#5-deploy-some-code).
+
+To create your web app using the portal:
+1.	Choose the **Subscription** where the Resource Group lives containing your Azure Arc resources
+2.	Choose the correct **Resource Group** that contains your Azure Arc resources (cluster, custom location, Kube environment, etc.)
+3.	Give your site a unique **Name**
+4.	Choose **.NET 5** for your **Runtime stack**
+5.	The **Operating System** will need to be **Linux** since App Service on Kubernetes is currently Linux only. 
+6.	Go to the **Region** drop down menu and choose your custom location under **Custom Locations (Preview)**. You may notice when you choose your custom location, the appended domain under the **Name** option is updated to include your Kube environment name and Arc region.  Where you typically would see *azurewebsites.net*, you now see *k4apps.io*.  Keep in mind the custom location is the abstracted layer on top of your Azure Arc enabled cluster that enables you to use Azure services.
+
+    ![grpc web]({{ site.baseurl }}/media/2021/07/grpc_arc_1.png)
+
+7.	Next, click **Review + create** to create your resource. Once your resource is created, you can view that it is deployed to your custom location by visiting the custom location resource.
+
+    ![grpc web]({{ site.baseurl }}/media/2021/07/grpc_arc_2.png)
```

Comment:
@jeffwmartinez FYI you need to tab-in any images or code snippets when using ordered lists. Otherwise they break the list and it looks like this:

1.
2.
3.
<image>
1.
2.

But if you tab-in the images or code snippets (lines 29, 33, 41, 48) then it will render correctly and keep the correct numbers